tez-plugins/tez-aux-services/src/main/java/org/apache/tez/auxservices/ShuffleHandler.java

```diff
@@ -1421,14 +1462,14 @@ protected ChannelFuture sendMapOutput(ChannelHandlerContext ctx, Channel ch,
       }
       ChannelFuture writeFuture;
       if (ch.pipeline().get(SslHandler.class) == null) {
-        final FadvisedFileRegion partition = new FadvisedFileRegion(spill,
+        final org.apache.tez.auxservices.FadvisedFileRegion partition = new org.apache.tez.auxservices.FadvisedFileRegion(spill,
```

Comment:
I don't think a fully qualified class name is needed here, as FadvisedFileRegion is in the same package

Reply:
ack